[2022-12-11 18:19:50,237] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: first_dag.write_diff_task manual__2022-12-11T18:18:52.504170+00:00 [queued]>
[2022-12-11 18:19:50,310] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: first_dag.write_diff_task manual__2022-12-11T18:18:52.504170+00:00 [queued]>
[2022-12-11 18:19:50,312] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-12-11 18:19:50,314] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-12-11 18:19:50,329] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-12-11 18:19:50,380] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): write_diff_task> on 2022-12-11 18:18:52.504170+00:00
[2022-12-11 18:19:50,417] {standard_task_runner.py:52} INFO - Started process 4363 to run task
[2022-12-11 18:19:50,488] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'first_dag', 'write_diff_task', 'manual__2022-12-11T18:18:52.504170+00:00', '--job-id', '1189', '--raw', '--subdir', 'DAGS_FOLDER/first-dag.py', '--cfg-path', '/tmp/tmpioce3l71', '--error-file', '/tmp/tmpk09w1ed2']
[2022-12-11 18:19:50,499] {standard_task_runner.py:80} INFO - Job 1189: Subtask write_diff_task
[2022-12-11 18:19:50,844] {task_command.py:369} INFO - Running <TaskInstance: first_dag.write_diff_task manual__2022-12-11T18:18:52.504170+00:00 [running]> on host 7e39590a7807
[2022-12-11 18:19:51,183] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=first_dag
AIRFLOW_CTX_TASK_ID=write_diff_task
AIRFLOW_CTX_EXECUTION_DATE=2022-12-11T18:18:52.504170+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-12-11T18:18:52.504170+00:00
[2022-12-11 18:19:51,323] {python.py:173} INFO - Done. Returned value was: None
[2022-12-11 18:19:51,913] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=first_dag, task_id=write_diff_task, execution_date=20221211T181852, start_date=20221211T181950, end_date=20221211T181951
[2022-12-11 18:19:52,062] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-12-11 18:19:52,396] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
